src/kibana/components/doc_table/components/table_header.js

```diff
@@ -12,10 +16,19 @@ define(function (require) {
         columns: '=',
         sorting: '=',
         indexPattern: '=',
+        type: '='
       },
       template: headerHtml,
       controller: function ($scope) {
-
+        $scope.downloadQueueManager = DownloadQueueManager;
+        $scope.captureSelectModalManager = CaptureSelectModalManager;
+        
+        if ($scope.type && ($scope.type === 'savedsearch')){
+            // sorry... had to grab panel name from parent (x8) scope :(
+            $scope.tableID = $scope.$parent.$parent.$parent.$parent.$parent.$parent.$parent.$parent.panel.id;
```

Comment:
would it be better to broadcast that id in an event that is picked up in this controller? not sure how well that will work, im just concerned with this breaking with an update